examples/scaling/main.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"image/png"
+	"log"
+	"os"
+
+	"github.com/asticode/go-astiav"
+)
+
+func main() {
+	var (
+		dstFilename string
+		dstWidth    int
+		dstHeight   int
+	)
+
+	flag.StringVar(&dstFilename, "output", "", "Output file name")
+	flag.IntVar(&dstWidth, "w", 0, "Destination width")
+	flag.IntVar(&dstHeight, "h", 0, "Destination height")
+	flag.Parse()
+
+	if dstFilename == "" || dstWidth <= 0 || dstHeight <= 0 {
+		fmt.Fprintf(os.Stderr, "Usage: %s -output output_file -w W -h H\n", os.Args[0])
```

Comment:
Could you use `log.Println()` instead with pattern `Usage: <binary path> -o <output path> -w <output width> -h <output height>` (followed by `return`)?